Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM continuumio/anaconda3:latest
```

Comment:
I would not use Anaconda Python here. IMHO `pip` installable packages should be based on the default Python installation (and specify the requirements in `setup.py`).